lib/jekyll/converters/smartypants.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module Converters
+    class Smartypants < Converter
+      safe true
+
+      priority :lowest
+
+      def matches(ext)
+        true
```

Comment:
!!!!!!!! :( this applies this to _every_ convertible file. definitely don't want this. what if it's JSON or something?